app/src/main/res/layout/fragment_host_editor.xml

```diff
@@ -25,6 +25,8 @@
 	>
 
 	<LinearLayout
+		android:focusable="true"
```

Comment:
Without these changes, when we clear focus on the font size EditText, focus automatically returns to the first focusable element, which is the quick connect container. It's just kind of janky UX to be fiddling with the font size and then have focus jump to a different section in the fragment.